package.json

```diff
@@ -149,6 +154,26 @@
       {
         "view": "deepcode.views.analysis",
         "contents": "DeepCode analyzed your code and found no issue! 🎉"
+      },
+      {
+        "view": "deepcode.views.actions",
+        "contents": "You are currently running DeepCode in manual mode.\n[Analyze now](command:deepcode.start)\n[Switch to auto-scan mode](command:deepcode.setmode?%5B%22auto%22%5D)",
```

Comment:
You are currently running DeepCode in manual mode. You are in control, no automated actions from our side.